.vscode/launch.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
this is fine to commit, but there isn't too much reason to have this as there isn't any special vscode config that we need, and the default config is fine